studio.config.json

```diff
@@ -3,43 +3,11 @@
     "dir": "",
     "menu": [
       "introduction",
-      [
-        "TailwindCSS",
-        [
-          "tailwind-showcases"
-        ]
-      ],
-      [
-        "CSS",
-        [
-          "css-showcases"
-        ]
-      ],
-      [
-        "Doc-site",
-        [
-          "layout",
-          "speedy-links"
-        ]
-      ],
-      [
-        "Components",
-        [
-          "box",
-          "clipboard",
-          "text",
-          "showcases",
-          "space",
-          "do-dont",
-          "search"
-        ]
-      ],
-      [
-        "Infra",
-        [
-          "doc-layout"
-        ]
-      ]
+      ["DESIGN TOKEN SHOWCASES", ["tailwind-showcases", "css-showcases"]],
+      ["COMPONENTS", ["do-dont", "note", "caption", "image-container", "row"]],
+      ["LAYOUTS", ["layout", "speedy-links"]],
```

Comment:
nitpick: it's not plural, this is layout related stuff, but there is only 1 layout